### DISCUSSION TITLE
Way to only show title of the target web page?

### DISCUSSION THREAD

**Discussion by geekjr** [jameslittle230/stork#140] [Q&A] [answered]
Hi. Is there a way to only show the title of the target webpage, not the matching keywords when displaying the results?

**Comment by jameslittle230** [comment, marked as answer]
Yes! You could
- set the `excerpts_per_result` config in your index’s TOML file to 0
- write CSS to hide the corresponding HTML elements you don’t want to see

Let me know if that’ll work.

**Reply by geekjr** [reply]
Thank you. It worked!
